package.json

```diff
@@ -74,15 +74,15 @@
   "license": "MIT",
   "main": "dist/index.js",
```

Comment:
```suggestion
  "main": "lib/index.js",
```

Reply:
Good catch. I thought it still generated a `dist` folder with only that but apparently not.